internal/pkg/cli/app_delete.go

```diff
@@ -255,9 +219,10 @@ func (o *deleteAppOpts) removeAppProjectResources() error {
 
 	o.spinner.Start(fmt.Sprintf("removing app %s resources from project %s", o.AppName, o.projectName))
 	if err := cfClient.RemoveAppFromProject(proj, o.AppName); err != nil {
-		o.spinner.Stop(log.Serrorf("removing app %s resources from project %s", o.AppName, o.projectName))
-
-		return err
+		if !stackSetNotExistErr(err) {
```

Comment:
I feel like the logic within `stackSetNotExistErr` should be handled inside the `RemoveAppFromProject` logic rather than clients handling that. Maybe return a `nil` error instead of trying to handle this scenario in client code?

Reply:
> I feel like the logic within `stackSetNotExistErr` should be handled inside the `RemoveAppFromProject` logic rather than clients handling that. Maybe return a `nil` error instead of trying to handle this scenario in client code?

Agree. Beyond that I think we need to refactoring the `cloudformation` package for the sake of testing, so I am going to work on another PR addressing that after this PR. Before the refactoring I'd rather not putting `stackSetNotExistErr` in it because `RemoveAppFromProject` might change.